modules/setting/database.go

```diff
@@ -16,9 +16,10 @@ import (
 )
 
 var (
-	// SupportedDatabases includes all supported databases type
-	SupportedDatabases = []string{"MySQL", "PostgreSQL", "MSSQL"}
-	dbTypes            = map[string]string{"MySQL": "mysql", "PostgreSQL": "postgres", "MSSQL": "mssql", "SQLite3": "sqlite3"}
+	// SupportedDatabaseTypes includes all XORM supported databases type, sqlite3 maybe added by `database_sqlite3.go`
+	SupportedDatabaseTypes = []string{"mysql", "postgres", "mssql"}
```

Comment:
Instead of duplicated val, can we just use string.lowercase on compare to sanitize input?

Reply:
Nope, they are different.

And `SupportedDatabaseTypes` differs with different builds (with or without sqlite3)

Reply:
ah I see - I still dont like this approach ... but It will do it

Reply:
We have to keep both of them.

 For setting system, we should keep `postgres`, otherwise it breaks settings of the existing users. 

For UI, we should show `PostgreSQL` (and `PostgreSQL` is really not suitable to be a setting value)

That's the best we can do now.